### PR TITLE
Git assignment - changes by Nila Le

### PR DESCRIPTION
**Pull request by nilale**
1. Forked repo to my account
2. Cloned the forked repo locally
3. Wrote a bash script that mirrored what we set up last week & committed the script
4. Added .gitkeep to script & committed change
5. Pushed new commits to original rep on GitHub

### CODE REVIEW

File: project_setup.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# A script to setup a bioinformatics project folder
+# Naupaka Zimmerman nzimmerman@usfca.edu
```
Comment (naupaka) [inline project_setup.sh]: This is not your name!

File: project_setup.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# August 30, 2017
+
+# Setup folder structure
+echo "Creating folder structure..."
+mkdir 2017-08-30_bioinformatics_project
```
Comment (naupaka) [inline project_setup.sh]: This is not today!

**Review by naupaka — CHANGES_REQUESTED** [review]
Some things need to be fixed.

**Review by nilale — COMMENTED** [review]
Requested changes fixed

**Pull request closed without merging**